packages/smart-wallet/src/walletFactory.js

```diff
@@ -237,6 +248,19 @@ export const prepare = async (zcf, privateArgs, baggage) => {
    */
   const makeSmartWallet = prepareSmartWallet(baggage, shared);
 
+  // One time repair for incarnation 2. We're adding WatchedPromises to allow
+  // wallets to durably monitor offer outcomes, but wallets that already exist
+  // need to be backfilled. This code needs to run once at the beginning of
+  // incarnation 2, and then shouldn't be needed again.
+  if (!baggage.has(UPGRADE_TO_INCARNATION_TWO)) {
+    trace('Wallet Factory upgrading to incarnation 2');
+
+    for (const wallet of walletsByAddress.values()) {
+      wallet.repairWalletForIncarnation2(upgradeToIncarnation2Key);
+    }
+    baggage.init(UPGRADE_TO_INCARNATION_TWO, 'done');
```

Comment:
"done" sounds like it will be read but it's not an invariant. Perhaps we need a common value or pattern for keys for which the value is meaningless. Even `null` or `undefined` would work here.

```suggestion
    baggage.init(UPGRADE_TO_INCARNATION_TWO, 'keypresent');
```

Reply:
> Perhaps we need a common value or pattern for keys for which the value is meaningless.

Apparently. I think you've made the same comment before (for a similar use), but with a different suggested name. I don't favor `'keypresent'`, but would be happy with 'defined', 'present', or 'plugh'. I don't think it takes long to verify that the value doesn't matter.

Reply:
>I don't think it takes long to verify that the value doesn't matter.

Not long, but not still cognitive load to search and remember.

not a big deal, resolving